src/select/types.js

```diff
@@ -134,6 +134,8 @@ export type PropsT = {
   maxDropdownHeight: string,
   /** Defines if multiple options can be selected. */
   multi: boolean,
+  /** Handle for accessing internal methods. */
+  methods?: React.ElementRef<*>,
```

Comment:
I think something like below may work without the deprecated `*` (I can't even find docs on the syntax anymore)

```
type ImperativeFnsT = {|
  setDropdownOpen: (boolean) => mixed,
|};

type PropsT = {|
  methods?: {| current: ImperativeFnsT | null |},
|};
```